libraries/Microsoft.Bot.Builder.AI.LUIS/V3/ListElement.cs

```diff
@@ -27,7 +27,7 @@ public ListElement()
         public ListElement(string canonicalForm, IList<string> synonyms = null)
         {
             CanonicalForm = canonicalForm;
-            Synonyms = synonyms;
+            Synonyms = synonyms ?? new List<string>();
```

Comment:
Could we use a ??= operator here instead of initialization of another List<string>?

Reply:
Hi @LeeParrishMSFT,
We don't think the ??= will work for these cases. It won't assign the value of the parameter if we have the initialization in the property, and if we remove the property initialization, it could fail if the constructor with no parameters is used.
We've been testing this [here](https://dotnetfiddle.net/RE3tc3).

Reply:
You are correct, I wasn't looking at it correctly. I'll resolved the comments on this and approve this PR.